components/studies/ui/src/main/resources/PhenoTips/StudyBindingClass.xml

```diff
@@ -72,7 +72,7 @@
     #end
     &lt;div class="form-template-option"&gt;
     &lt;label&gt;&lt;input type="radio" name="${prefix}${name}" value="${escapetool.xml($services.model.serialize($s.documentReference, 'default'))}" #if ("$!{value}" == "${s.documentReference}") checked="checked" #end/&gt; $escapetool.xml($configTitle)&lt;/label&gt;
-    &lt;p class="xHint"&gt;$!{escapetool.xml($s.content)}&lt;/p&gt;
+    &lt;div class="xHint"&gt;$!s.getRenderedContent()&lt;/div&gt;
```

Comment:
I wonder if a "plainitization" of the rendered content would be the best approach. I agree that the raw content isn't right, but I also don't think that we need the full HTML in here. Here's the difference between the two:

<img width="794" alt="image" src="https://user-images.githubusercontent.com/88663/56754395-eb112e00-675a-11e9-9455-54fe150e66ca.png">

```suggestion
    &lt;div class="xHint"&gt;$!{escapetool.xml($s.getRenderedContent($services.rendering.resolveSyntax('plain/1.0'))).replaceAll('\n', '&lt;br /&gt;')}&lt;/div&gt;
```

Reply:
Sounds good, that would prevent weird font sizes and layouts caused by elements such as headings/img .